templates/community-viz/src/index.js

```diff
@@ -1,38 +1,122 @@
 const dscc = require('@google/dscc');
 
+
+
 // import the local data
 import * as local from './localData.js';
 
 // change this to 'true' for local development
 // change this to 'false' before deploying
-const LOCAL = false;
+const LOCAL = true;
```

Comment:
So we want to default to a local install now?

Reply:
yes